silkworm/rpc/core/evm_debug.hpp

```diff
@@ -111,7 +111,7 @@ class DebugTracer : public EvmTracer {
     const char* const* opcode_names_ = nullptr;
     const evmc_instruction_metrics* metrics_ = nullptr;
     std::stack<std::int64_t> start_gas_;
-    std::unique_ptr<CallFixes> call_fixes_;
+    std::unique_ptr<FixCallGasInfo> fix_call_gas_info_;
```

Comment:
It's probably clearer and potentially also faster using `std::optional` instead of `std::unique_ptr` here.
You can do in next PR